bin/fulltext.py

```diff
@@ -1,9 +1,29 @@
+import os
 import multiprocessing
+from argparse import ArgumentParser
+from arxiv_public_data.fulltext import convert_directory_parallel
+from arxiv_public_data.config import DIR_BASE, DIR_FULLTEXT, DIR_OUTPUT
 from arxiv_public_data import s3_bulk_download
+from arxiv_public_data.s3_bulk_download import call
 
 if __name__ == '__main__':
-    import sys
-    processes = multiprocessing.cpu_count() if len(sys.argv) <= 1 else int(sys.argv[1])
 
-    manifest = s3_bulk_download.get_manifest()
-    s3_bulk_download.process_manifest_files(manifest, processes=processes)
+    parser = ArgumentParser(description="""Convert all pdfs contained in a directory and its sub-directories into txt 
+    files""")
+    parser.add_argument("-N", type=int, default=multiprocessing.cpu_count(), help="OPTIONAL number of CPUs, default "
+                                                                                  "all available")
+    parser.add_argument("--PLAIN_PDFS", type=bool, default=False,
```

Comment:
If you remove `type=bool` and add `action="store_true"`, then the flag will require no arguments and its presence or absence will determine its value. Then the usage of the tool is simpler: `bin/fulltext.py -N 20 --PLAIN_PDFS`.

Reply:
That's neat! Thanks

Reply:
Done